master-qa-14755

austinchiang opened the pull request:
https://issues.liferay.com/browse/LRQA-14755

austinchiang committed:
LRQA-14755 paths

austinchiang committed:
LRQA-14755 macros

liferay-continuous-integration commented:
<h1>All tests PASSED!</h1><p>Build Time: 12 minutes 18 seconds</p><h1>See the following links for more information.</h1><ul><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest(master)/388/">https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest(master)/388/</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-start(master)/388/">https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-start(master)/388/</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/381/">https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/381/</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-portalweb(master)/52/">https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-portalweb(master)/52/</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-stop(master)/381/">https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-stop(master)/381/</a></li></ul>

Commit:
LPS-52679 Create new module for common CXF infrastructure

There is CXF infrastructure which is common for both JAX-WS and JAX-RS.
It seems wise to extract it to a different bundle so, in the future, the
JAX-WS and JAX-RS stuff can be used independently from the other.

Commit:
LPS-52679 Camel case

Commit:
LPS-52679 Use configuration in the config name?

Commit:
LPS-52679 Don't expose lifecycle related methods

Commit:
LPS-52679 SF

Commit:
LPS-52679 SF

marcellustavares committed:
LPS-54988 Initial commit for DDL Form portlet

marcellustavares committed:
LPS-54988 Add implementation classes

marcellustavares committed:
LPS-54988 Add views

marcellustavares committed:
LPS-54988 Language.properties

Commit:
LPS-54946 Coloring inconsistency of required fields of a web content structure

Commit:
LPS-52679 Extract common configuration to common bundle

Commit:
LPS-52679 Copy file

Commit:
LPS-52679 Move Servlet infrastructure to common module

marcellustavares committed:
LPS-54988 Add WebKey and utility method to MVCPortlet

pat270 committed:
LPS-54946 Source Formatting

Commit:
LPS-55112 update tests

Commit:
LPS-55112 access MB via MBUtil

Commit:
LPS-55112 do not inject MB

Commit:
LPS-55112 do not inject MB

Commit:
LPS-55112 use factory to create single destination sender

Commit:
LPS-55112 add factory for constructing single destination message senders

Commit:
LPS-55112 use factory to create new message sender

Commit:
LPS-54946 - Source formatting

jkappler committed:
LPS-55119 Added missing mvcPath parameter to the form

arboliveira committed:
LPS-55083 Elasticsearch type mappings: merge default config into liferay 

https://issues.liferay.com/browse/LPS-55083

Commit:
LPS-55112 messaging should init earlier

Commit:
LPS-55147 If the parent message is the default one, we do not need to propagate permissions. Also, passing default parent message id to propagatePermssion method  can lead NoSuchMessage exception.

Commit:
LPS-55147 Refactor the method getSubjectFromMail. Since we already get parent message before calling it, we can directly get subject from the message. Also, if the mail message contains sub-domain, the getSubjectFromMail cannot get correct message ID.

brianwulbern committed:
LRQA-15157 Ignore while we investigate LPS-55120

evannagayama committed:
LRQA-15157 SF

sharonchoi commented:
Pushed tests back to AChiang:
-Please remove fixes for "Draft" to "draft" as that is a workflow UI bug
 @austinchiang

cgoncas committed:
LPS-55159 There are no JumpHolder or SwithHolder in new version, so we are going to use those classes from cobertura 1.9.4.1

cgoncas committed:
LPS-55159 Add copyright and author

cgoncas committed:
LPS-55159 SF

marcellustavares committed:
LPS-52679 Add portlet description

cgoncas committed:
LPS-55159 Use the new class in the visitors

shuyangzhou committed:
LPS-55159 Repackage

shuyangzhou committed:
LPS-55159 Auto SF

shuyangzhou committed:
LPS-55159 SF

Ithildir committed:
LRQA-14419 Override properties

hhuijser committed:
LPS-55147 let's actually assign a value to subject...

shuyangzhou committed:
LPS-55159 Generate

shuyangzhou committed:
Revert "LPS-54998 SF"

This reverts commit a5ead21f07a084e921a291a6302a3c49f8ff302f.

shuyangzhou committed:
Revert "LPS-54998 SF"

This reverts commit ed19fa37f05644d93a02d46935623b9922385f9e.

shuyangzhou committed:
Revert "LPS-54998 - add method to PortalWebResourcesUtil to check file availability"

This reverts commit 0d88f1099000ebb32812ee3ea44cdfc90b5efe87.

Commit:
LPS-52679 Make Symbolic and packages be named the same

ealonso committed:
LPS-55119 SF

jbalsas committed:
LPS-55172 Update switch type to toggle-switch

pat270 committed:
LPS-55172 Update toggle-switch CSS

pat270 committed:
LPS-55172 Add toggle-card CSS

jbalsas committed:
LPS-55172 Update input taglib to support toggle-card type

Commit:
LPS-55083 add missing fields

marcellustavares committed:
LPS-54988 Create a default page with empty title for the default DDMFormLayout

marcellustavares committed:
LPS-54988 Fix copy constructor

marcellustavares committed:
LPS-54988 Fix paths

jbalsas committed:
LPS-55172 SF css

jbalsas committed:
LPS-55172 SF

Commit:
LPS-55112 prepare MessageBusUtil so that it can be modularized

Commit:
LPS-55112 use factory to construct single destination message sender

natecavanaugh committed:
LPS-55172 - Rename to fit our patterns

Commit:
LPS-55041 Edit language key text

brianchandotcom committed:
LPS-55083 SF

Commit:
LPS-55112 fix test

brianchandotcom committed:
LPS-55083 Sort

natecavanaugh committed:
LPS-55172 - Removing duplicate code

brianchandotcom committed:
LPS-55112 SF

brianchandotcom committed:
LPS-55112 SF

brianchandotcom committed:
LPS-55112 Sort

brianchandotcom committed:
LPS-55112 No need for package

brianchandotcom committed:
LPS-55112 Proper syntax

brianchandotcom committed:
LPS-55112 SF

brianchandotcom committed:
LPS-55112 It's already static

Commit:
LPS-53947 Re-add text removed in ec137190660fd9142ff1dcfb5b9fa59a5c3e1dbe

brianchandotcom committed:
LPS-55112 SF

Commit:
LPS-53947 Edit Javadoc

brianchandotcom committed:
LPS-55112 SF

brianchandotcom committed:
LPS-55172 SF

Ithildir committed:
LRQA-14419 Declare Ant dependency

Ithildir committed:
LRQA-14419 List property values even if overriden by system properties

marcellustavares committed:
LPS-54988 Use import.shared for now

brianchandotcom committed:
LPS-52679 Rename

brianchandotcom committed:
LPS-52679 Can I sort this?

brianchandotcom committed:
LPS-52679 Can I sort this?

brianchandotcom committed:
LPS-52679 fix Eclipse

brianchandotcom committed:
LPS-52679 Not thrown

Ithildir committed:
LPS-55187 Gradle plugin

brianchandotcom committed:
LPS-52679 Fix warning for now

brianchandotcom committed:
LPS-52679 sort

alee8888 committed:
LRQA-15073 macros

alee8888 committed:
LRQA-15073 tests

michaelhashimoto committed:
LRQA-14419 Load 'jserrorcollector' as a class resource instead of a file

brianchandotcom committed:
LPS-52679 as used

michaelhashimoto committed:
LRQA-14419 Read resource files using a ClassLoader instead of reading the files directly

natecavanaugh committed:
LPS-54987 - Source formatting

brianchandotcom committed:
LPS-52679 SF

brianchiu committed:
LRQA-15157 SF

brianchandotcom committed:
LRQA-14419 Simplify

brianchandotcom committed:
LRQA-14419 Simplify

brianchandotcom committed:
LRQA-14419 SF

brianchandotcom committed:
LRQA-14419 SF

brianchandotcom committed:
LPS-55187 SF

brianchandotcom committed:
LPS-54988 not a proper noun like Bruno or Brian or Marcellus

brianchandotcom committed:
LPS-54988 use the word "application" whenever possible

brianchandotcom committed:
LPS-54988 Regen

brianchandotcom committed:
LPS-54988 Lang keys

brianchandotcom committed:
LPS-55041 lang keys

Ithildir committed:
LPS-55187 Always use latest release

brianchandotcom committed:
Prep for next version

brianchandotcom committed:
Prep for next version

michaelhashimoto committed:
LRQA-14419 Remove unneeded "/" from resource loaded files

brianchandotcom committed:
LRQA-14419 SF

brianchandotcom committed:
LRQA-14419 prepare for next version

sharonchoi commented:
No test changes needed :)

sharonchoi commented:
Pushed to BChiu:
https://github.com/brianchiu/liferay-portal/pull/212
